axelrod/tests/integration/test_filtering.py

```diff
@@ -35,15 +33,18 @@ def test_boolean_filtering(self):
         min_memory_depth=integers(min_value=1, max_value=10),
         max_memory_depth=integers(min_value=1, max_value=10),
         memory_depth=integers(min_value=1, max_value=10),
+        strategies=strategy_lists(min_size=20, max_size=20),
     )
     @example(
         min_memory_depth=float("inf"),
         max_memory_depth=float("inf"),
         memory_depth=float("inf"),
+        strategies=strategy_lists(min_size=20, max_size=20),
```

Comment:
I believe this should be an actual list of strategies here for the `@example`. To do this properly with hypothesis I believe we should remove the example case here, and write a second test OR we could go with just `strategies=short_run_time_strategies` (the correct import will be needed at the top). I've run this locally and it seems fine without a large speedup.

Reply:
It's strange because these tests are running fine for me locally but not on Travis, and we didn't explicitly list strategies in this example previously.

Reply:
Strange that it runs locally... Perhaps a slightly different version of hypothesis has implemented the ability to use strategies inside examples... Would be neat if it did...

Previously I think we were just using `all_strategies` from the global space.